helpers/changelog/main.go

```diff
@@ -24,6 +29,11 @@ func main() {
 			}
 			in = true
 		}
-		fmt.Println(line)
+
+		_, err := fw.WriteString(line + "\n")
+		if err != nil {
+			panic(err)
+		}
 	}
+	fw.Sync()
```

Comment:
I've never used this method before. I don't think it's necessary.
But you can leave it as is if you want.